<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignalWin</class>
 <widget class="QWidget" name="SignalWin">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1719</width>
    <height>877</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>信号灯编辑</string>
  </property>
  <widget class="QTableWidget" name="signalTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>941</width>
     <height>151</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>941</width>
     <height>0</height>
    </size>
   </property>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>subtype</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>roadid</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>s</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>t</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>h</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>reference</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>信号灯编辑：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signalAddBT">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signalDeleteBT">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveSignalBT">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="objectTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>1721</width>
     <height>281</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>sr</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>st</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>sh</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>s</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>t</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>zOffset</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>validLength</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>orientation</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>length</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>width</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>height</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>hdg</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>pitch</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>roll</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="manualCBB">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>否</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>是</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>342</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>是否手工处理：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>sr：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="srLE">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="stLE">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>st：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>380</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>type：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="typeLE">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>380</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>380</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>name：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLE">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>380</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>380</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>id：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLE">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>380</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="sLE">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>380</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>380</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>s：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tLE">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>380</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>380</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>t：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="zOffsetLE">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>410</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>2.0000000000000000e-02</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>zOffset：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="validLengthLE">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>410</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0000000000000000e+00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>410</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>validLength：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="orientationLE">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>410</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>none</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>410</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>orientation：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>410</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>length：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lengthLE">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>410</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0000000000000000e+00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>410</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>width：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="widthLE">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>410</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1.0000000000000000e+00</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="heightLE">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>410</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0000000000000000e+00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>410</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>height：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hdgLE">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0000000000000000e+00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>hdg：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>440</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>pitch：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pitchLE">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>440</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0000000000000000e+00</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rollLE">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>440</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0000000000000000e+00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>440</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>roll：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="objectAddBT">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>添加行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="objectDeleteBT">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>520</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>删除行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveObjectBT">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>520</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>380</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>sh：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="shLE">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>380</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
